src/pages/calendar/components/Section.tsx

```diff
@@ -59,9 +59,15 @@ export function SectionInfo({
   seat,
   meetingTimes,
 }: SectionInfoProps): JSX.Element {
-  const isASYNC = additionalNotes?.indexOf('asynchronous') !== -1;
+  const isASYNC = additionalNotes?.indexOf('This course will be offered fully online and asynchronous') !== -1;
+  const isBLENDED = additionalNotes?.indexOf('a mix of “real-time” and asynchronous sessions') !== -1;
   const isSENG = additionalNotes?.indexOf('Reserved for BSENG students') !== -1;
   const isCSC = additionalNotes?.indexOf('Reserved for students in a Computer Science program') !== -1;
+  const isENGR = additionalNotes?.indexOf('Restricted to BEng students only') !== -1;
+  const isENGR_CSC = additionalNotes?.indexOf('Restricted to students in the Faculty of Engineering') !== -1;
```

Comment:
to avoid mixing camel case and underscores, maybe a good name for this stuff is `isENGRorCSC`, up to you though